portals/knowledge-management/src/Pages/ViewArticle.vue

```diff
@@ -63,6 +70,9 @@
     const article = ref(null);
     const portalViewCount = ref(0);
     const portalViewCountFlag = ref(false);
+    const feedback = ref(null);
+    const featureFlag = ref(null);
+    const { user, setguestId } = useAuthStore();
```

Comment:
I don't think we need anything about the user or guest Id here and it should be removed

Reply:
code removed.